server/base/src/test/java/org/apache/accumulo/server/util/time/SimpleTimerTest.java

```diff
@@ -70,35 +69,50 @@ public void run() {
     }
   }
 
-  @Test
+  @Test(timeout = 5000)
   public void testOneTimeSchedule() throws InterruptedException {
     AtomicInteger i = new AtomicInteger();
     Incrementer r = new Incrementer(i);
     t.schedule(r, DELAY);
     Thread.sleep(DELAY + PAD);
-    assertEquals(1, i.get());
+    while (true) {
+      if (i.get() == 1) {
+        break;
+      }
+      Thread.sleep(PAD);
+    }
```

Comment:
Just realized that you can simplify this to `while (i.get() != 1) { sleep }`; you can do similar things for the other tests as well.

Reply:
Was considering that as well. I will add this change.